testutil/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/regen-network/regen-ledger/testutil
```

Comment:
As per https://github.com/regen-network/regen-ledger/pull/322#discussion_r613348641, we don't need this folder anymore.

We should instead create a `./app/modules_test.go` which imports x/* testsuites, and runs them.